repose-aggregator/docs/src/asciibinder/services/http-logging.adoc

```diff
@@ -0,0 +1,322 @@
+```
+
+| outboundResponseStatusCode
+| Integer
+| The status code of the outbound response.
+For example, for a response with the status line `HTTP/1.1 200 OK`, this value would be `200`.
+
+| outboundResponseReasonPhrase
+| String
+| The reason phrase of the outbound response.
+For example, for a response with the status line `HTTP/1.1 200 OK`, this value would be `OK`.
+
+| outboundResponseContentLength
+| Integer
+| The size of the outbound response body in bytes.
+This value is taken from the `Content-Length` header of the response.
+For example, for a response with the header line `Content-Length: 1234`, this value would be `1234`.
+
+| outboundResponseBytesWritten
+| Integer
+| The number of bytes of the outbound response body written.
+
+| timeRequestReceived
+| https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html[Instant]
+| The time at which the inbound request was received.
+Evaluating this variable in an expression (i.e., `{{ timeRequestReceived }}`) will result in the value of https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#toString--[Instant.toString] being rendered.
+
+| timeToHandleRequest
+| https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html[Duration]
+| The amount of time between receiving an inbound request and sending an outbound response.
+Evaluating this variable in an expression (i.e., `{{ timeToHandleRequest }}`) will result in the value of https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#toString--[Duration.toString] being rendered.
+This value can be rendered as the amount of time in milliseconds by calling the https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#toMillis--[Duration.toMillis] method (i.e., `{{ timeToHandleRequest.toMillis }}`).
+
+| timeInOriginService
+| https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html[Duration]
+| The amount of time between sending an outbound request to the origin service and receiving an inbound response.
+Evaluating this variable in an expression (i.e., `{{ timeInOriginService }}`) will result in the value of https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#toString--[Duration.toString] being rendered.
+This value can be rendered as the amount of time in milliseconds by calling the https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html#toMillis--[Duration.toMillis] method (i.e., `{{ timeToHandleRequest.toMillis }}`).
+
+| localIpAddress
+| String
+| The Internet Protocol (IP) address of the interface on which the inbound request was received.
+
+| remoteIpAddress
+| String
+| The Internet Protocol (IP) address of the client or last proxy that sent the inbound request.
+
+| remoteHost
+| String
+| The fully qualified name of the client or the last proxy that sent the inbound request.
+
+| traceId
+| String
+| The ID associated with the inbound request used for tracing purposes.
+This value will match the value from the decoded `X-Trans-Id` header.
+
+| extensions
```

Comment:
Where are all the auth values? They were supposed to be explicitly exposed.

Reply:
Ah, I seem to have missed that. I see the `username` called out in the acceptance criteria, but are there any other auth values you want to expose explicitly? I will add whatever is missing but desired.

Reply:
I think we wanted impersonator info as well